incident/incident.go

```diff
@@ -25,26 +25,42 @@ import (
 
 // Reporter describes a mechanism for recording compliance incidents.
 type Reporter interface {
-	// Log records an incident with the given parameters.
+	// LogUpdate records a non-violation incident with the given parameters.
 	// The baseURL, summary and category fields should be stable across
-	// multiple similar incidents to allow aggregation.  Information that
+	// multiple similar incidents to allow aggregation. Information that
 	// varies between instances of the 'same' incident should be included in
 	// the fullURL or details field.
-	Log(ctx context.Context, baseURL, summary, category, fullURL, details string)
-	Logf(ctx context.Context, baseURL, summary, category, fullURL, detailsFmt string, args ...interface{})
+	// isViolation helps distinguish between violation reports and state-updates
+	// including root changes.
```

Comment:
Can remove this sentence about `isViolation` now.

Reply:
Done